akka-docs/src/main/paradox/typed/routers.md

```diff
@@ -73,7 +73,7 @@ Java
 
 ## Routing strategies
 
-There are two different strategies for selecting what routee a message is forwarded to that can be selected
+There are three different strategies for selecting what routee a message is forwarded to that can be selected
```

Comment:
Shouldn't we say `which` instead of `what`? 

```suggestion
There are three different strategies for selecting which routee a message is forwarded to that can be selected
```

Furthermore, I think the whole sentence would read better if it was:

> There are three different strategies for selecting which routee a message is forwarded to that can be chosen before spawning the router: